CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - peer/direct: peer connections were closed even if they were still in use.
 - configuration: clarify error message for the special case of attempting to use a peer list
   updater when none have been registered.
+- protoplugin: GoType conventions aligned with gogo/protobuf.  
```

Comment:
I am not very convinced about this changelog entry. Should it be more:
- protoplugin: generated code follows go type conventions from gogo/protobuf.